src/main/java/uk/gov/hmcts/reform/profilesync/schedular/UserProfileSyncJobScheduler.java

```diff
@@ -41,56 +41,61 @@
     @Value("${loggingComponentName}")
     protected String loggingComponentName;
 
+
     @Scheduled(cron = "${scheduler.config}")
     public void updateIdamDataWithUserProfile() {
 
         String searchQuery = "(roles:pui-case-manager OR roles:pui-user-manager OR roles:pui-organisation-manager OR "
                 + "roles:pui-finance-manager) AND lastModified:>now-";
-
-
-        SyncJobConfig syncJobConfig =  syncConfigRepository.findByConfigName("firstsearchquery");
+        LocalDateTime startTime = LocalDateTime.now();
+        SyncJobConfig syncJobConfig =  profileSyncConfigRepository.findByConfigName("firstsearchquery");
 
         String configRun =  syncJobConfig.getConfigRun().trim();
-
-        log.info("{}:: Job needs to be run From Last::hours::", loggingComponentName, configRun);
+        ProfileSyncAudit  syncAudit = new ProfileSyncAudit();
+        log.info("{}:: Job needs to be run From Last::hours::{}" + configRun, loggingComponentName);
```

Comment:
please remove + not needed with log.info

Reply:
without that loggers not printing proper in the master . that is the fix for loggers issue